packages/react-native/scripts/honeybadger-upload-sourcemaps.sh

```diff
@@ -0,0 +1,305 @@
+#!/bin/bash
+
+# ----------------------------------------------------------------------------
+#
+# Generate and upload sourcemaps to Honeybadger
+#
+# USAGE:
+# npx honeybadger-upload-sourcemaps [--no-hermes] --apiKey <project-api-key> --revision <build-revision>
+#
+# Hermes is enabled by default in React Native projects as of React Native 0.70.
+# This script assumes that Hermes is being used. If you are not using Hermes in 
+# your React Native project, use the "--no-hermes" flag.
+#
+# ----------------------------------------------------------------------------
+
+USAGE="npx honeybadger-upload-sourcemaps [--no-hermes] --apiKey <project-api-key> --revision <build-revision>"
```

Comment:
Can you make the "upload to HB" optional? Something like:
```suggestion
USAGE="npx honeybadger-upload-sourcemaps [--no-hermes] [--skip-upload] --apiKey <project-api-key> --revision <build-revision>"
```

Reply:
Added.